app/server/tests/integration/session.test.ts

```diff
@@ -114,13 +114,14 @@ describe("Session id integration", () => {
 
     it("can create a friendly label for a session", async () => {
         const url = `apps/day1/sessions/${sessionId}/friendly`;
+        const friendlyIdRegex = /^([a-z]+-)+[a-z]+$/;
```

Comment:
unrelated change, fixes mrc-3643 which was failing periodically (see ticket for details)